<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui::SignalForm</class>
 <widget class="QWidget" name="gui::SignalForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="title">
       <property name="text">
        <string>Signalgenerator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="signalTypeBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="secondLabel">
       <property name="text">
        <string>Frequenzmultiplikator
f√ºr Kanal B (Pin 10)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="frequencyBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="frequencySlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="frequencyLabel">
       <property name="text">
        <string>Frequenz
Kanal A (Pin 14)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="secondBox"/>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="secondSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="amplitudeBox"/>
     </item>
     <item row="0" column="2">
      <widget class="QSlider" name="amplitudeSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="amplitudeLabel">
       <property name="text">
        <string>Amplitude
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
